deploy/grafana-dashboard/utils/color_mapping.py

```diff
@@ -19,3 +21,9 @@ def color_mapping(name, color):
 
 
 grey_all_mapping = color_mapping("all", "#4c4b5a")
+
+color_mapping_environments = {}
```

Comment:
Do you see any chance to decide for colors dynamically, e.g. based on the name? Let's consider, we would have a fixed color palette with ~20 sorted values. Then, we could just sort the names and map one color to each name / ID. This would be stable as long as the name is not changed and would still be dynamic so that new environments don't need to get reflected in the code. 

Just asking, probably not achievable (because you're making the color assignment here in Python before you know how many and which environments we have...)

Reply:
> you're making the color assignment here in Python before you know how many and which environments we have...

Jup, I'm struggling overcome that. What do you think about the following solution:
- In this issue we read the environments from a text file and map them to `a fixed color palette with ~20 sorted values`.
- In another issue we create an ansible playbook that
  - Fetches the current environments from Poseidon and writes the text file
  - Generates and deploys the current dashboard

Reply:
Sounds good, let's do that!

Reply:
See [3b168aa](https://github.com/openHPI/poseidon/pull/245/commits/3b168aaf905cf0f943fbd629124e4886a659c622) and #255

Reply:
Nice, I like it, thanks! I just have some color suggestions:

Current version:

<img width="764" alt="Bildschirm­foto 2022-11-26 um 13 23 47" src="https://user-images.githubusercontent.com/7300329/204089069-ca69ee69-da4e-4d30-a86d-ffa2f0f55e58.png">

R ColorBrewer:

<img width="767" alt="Bildschirm­foto 2022-11-26 um 13 27 38" src="https://user-images.githubusercontent.com/7300329/204089072-7c22c72f-d7b3-49f2-87b9-09fce39fc440.png">

Sorted R ColorBrewer:

<img width="758" alt="Bildschirm­foto 2022-11-26 um 13 28 27" src="https://user-images.githubusercontent.com/7300329/204089073-87b598dc-f930-4266-b62a-0f2b9dbda5a3.png">


I deployed that to "Poseidon autogen Seb", so feel free to have a look :)